server/shared/src/main/scala/org/http4s/server/middleware/Throttle.scala

```diff
@@ -53,40 +56,50 @@ object Throttle {
       def takeToken: G[TokenAvailability] = fk(t.takeToken)
     }
 
+    private def getTime[F[_]: Temporal] =
+      Temporal[F].monotonic.map(_.toNanos)
+
     /** Creates an in-memory [[TokenBucket]].
       *
-      * @param capacity the number of tokens the bucket can hold and starts with.
+      * @param capacity    the number of tokens the bucket can hold and starts with.
       * @param refillEvery the frequency with which to add another token if there is capacity spare.
       * @return A task to create the [[TokenBucket]].
       */
     def local[F[_]](capacity: Int, refillEvery: FiniteDuration)(implicit
         F: Temporal[F]
-    ): F[TokenBucket[F]] = {
-      def getTime = F.monotonic.map(_.toNanos)
```

Comment:
Ah, I see, you moved it from here. Actually if this became a `val` then it would be slightly more efficient than as a `private def`.

Reply:
Yep, good shout